vm-virtio/src/vsock/device.rs

```diff
@@ -394,6 +398,12 @@ pub struct Vsock<B: VsockBackend> {
     path: PathBuf,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct VsockState {
+    pub avail_features: u64,
+    pub acked_features: u64,
```

Comment:
Maybe we need something like a VirtioState to handle things like this.

Reply:
Yes I'll open an issue to factorize every virtio device state through a common VirtioState structure.

Reply:
https://github.com/cloud-hypervisor/cloud-hypervisor/issues/1211